Tests/LibWeb/Text/expected/wpt-import/url/url-constructor.any.txt

```diff
@@ -0,0 +1,881 @@
+Pass	Parsing: <blob:ws://example.org/> without base
+Pass	Parsing: <blob:wss://example.org/> without base
+Pass	Parsing: <blob:http%3a//example.org/> without base
+Pass	Parsing: <http://0x7f.0.0.0x7g> without base
+Pass	Parsing: <http://0X7F.0.0.0X7G> without base
+Pass	Parsing: <http://[::127.0.0.0.1]> without base
+Pass	Parsing: <http://[0:1:0:1:0:1:0:1]> without base
+Pass	Parsing: <http://[1:0:1:0:1:0:1:0]> without base
+Pass	Parsing: <http://example.org/test?"> without base
+Pass	Parsing: <http://example.org/test?#> without base
+Pass	Parsing: <http://example.org/test?<> without base
+Pass	Parsing: <http://example.org/test?>> without base
+Pass	Parsing: <http://example.org/test?⌣> without base
+Pass	Parsing: <http://example.org/test?%23%23> without base
+Pass	Parsing: <http://example.org/test?%GH> without base
+Pass	Parsing: <http://example.org/test?a#%EF> without base
+Pass	Parsing: <http://example.org/test?a#%GH> without base
+Pass	Parsing: <a> without base
+Pass	Parsing: <a/> without base
+Pass	Parsing: <a//> without base
+Pass	Parsing: <test-a-colon.html> against <a:>
+Pass	Parsing: <test-a-colon-b.html> against <a:b>
+Pass	Parsing: <test-a-colon-slash.html> against <a:/>
+Pass	Parsing: <test-a-colon-slash-slash.html> against <a://>
+Pass	Parsing: <test-a-colon-slash-b.html> against <a:/b>
+Pass	Parsing: <test-a-colon-slash-slash-b.html> against <a://b>
+Pass	Parsing: <http://example.org/test?a#b c> without base
+Pass	Parsing: <non-spec://example.org/test?a#b c> without base
+Pass	Parsing: <non-spec:/test?a#b c> without base
+Pass	Parsing: <10.0.0.7:8080/foo.html> against <file:///some/dir/bar.html>
+Pass	Parsing: <a!@$*=/foo.html> against <file:///some/dir/bar.html>
+Pass	Parsing: <a1234567890-+.:foo/bar> against <http://example.com/dir/file>
+Pass	Parsing: <file://a­b/p> without base
+Pass	Parsing: <file://a%C2%ADb/p> without base
+Pass	Parsing: <file://loC���U+dc00���U+dc0b���U+dc07���U+dc28���U+dc2c���U+dc2d/usr/bin> without base
+Pass	Parsing: <file://­/p> without base
+Pass	Parsing: <file://%C2%AD/p> without base
+Pass	Parsing: <file://xn--/p> without base
+Pass	Parsing: <#link> against <https://example.org/##link>
+Pass	Parsing: <non-special:cannot-be-a-base-url- ~> without base
+Pass	Parsing: <non-special:cannot-be-a-base-url-!"$%&'()*+,-.;<=>@[\]^_`{|}~@/> without base
+Pass	Parsing: <https://www.example.com/path{path.html?query'=query#fragment<fragment> without base
+Pass	Parsing: <https://user:pass[@foo/bar> against <http://example.org>
+Pass	Parsing: <foo:// !"$%&'()*+,-.;<=>@[\]^_`{|}~@host/> without base
+Pass	Parsing: <wss:// !"$%&'()*+,-.;<=>@[]^_`{|}~@host/> without base
+Pass	Parsing: <foo://joe: !"$%&'()*+,-.:;<=>@[\]^_`{|}~@host/> without base
+Pass	Parsing: <wss://joe: !"$%&'()*+,-.:;<=>@[]^_`{|}~@host/> without base
+Pass	Parsing: <foo://!"$%&'()*+,-.;=_`{}~/> without base
+Pass	Parsing: <wss://!"$&'()*+,-.;=_`{}~/> without base
+Fail	Parsing: <foo://host/ !"$%&'()*+,-./:;<=>@[\]^_`{|}~> without base
```

Comment:
note for maintainers - these two tests conflicts with the percent encoding merge request